.github/actions/prepare.sh

```diff
@@ -37,17 +37,20 @@ done
 for package in ${DEPENDENT_PACKAGES}
 do
     echo "===> Searching for dependent package: ${package}"
-    packages=$(echo "${DEPENDENCY_LIST}" | grep -w "${package}" | grep -o ".*:" | tr ':' ' ' | sort -u | tr '\n' ' ')
+    packages=$(echo "${DEPENDENCY_LIST}" | grep " ${package} " | grep -o ".*:" | tr ':' ' ' | sort -u | tr '\n' ' ')
     echo "===> Found: ${packages}"
     SPK_TO_BUILD+=${packages}
 done
 
 # fix for packages with different names
 if [ "$(echo ${SPK_TO_BUILD} | grep -ow nzbdrone)" != "" ]; then
-    SPK_TO_BUILD+=" sonarr3"
+    SPK_TO_BUILD=$(echo "${SPK_TO_BUILD}" | tr ' ' '\n' | grep -v "nzbdrone" | tr '\n' ' ')" sonarr3"
 fi
 if [ "$(echo ${SPK_TO_BUILD} | grep -ow python)" != "" ]; then
-    SPK_TO_BUILD+=" python2"
+    SPK_TO_BUILD=$(echo "${SPK_TO_BUILD}" | tr ' ' '\n' | grep -v "python" | tr '\n' ' ')" python2"
+fi
+if [ "$(echo ${SPK_TO_BUILD} | grep -ow ffmpeg)" != "" ]; then
+    SPK_TO_BUILD=$(echo "${SPK_TO_BUILD}" | tr ' ' '\n' | grep -v "ffmpeg" | tr '\n' ' ')" ffmpeg4"
```

Comment:
If I get this right, you're removing all `ffmpeg{456}` from the list but only adding `ffmpeg4` at the end of list?  That would then by default disable `ffmpeg{56}` builds, no?

Reply:
No, this is not the case, it just renames ffmpeg to ffmpeg4. (`-w ffmpeg` is a full word filter and does not match any of ffmpeg4, ffmpeg5 and ffmpeg6).

The error that is fixed with this is the following:
The modification of a file in the spk/ffmpeg4 folder triggers the build of ffmpeg package (not ffmpeg4) and I saw errors saying the folder spk/ffmpeg does not exist.
This custom handling is required for all packages where the package name does not match the spk folder (see lines above for python2 and sonarr3).

Reply:
Makes full sense, another nice catch :)